medusa/name_parser/parser.py

```diff
@@ -117,35 +119,52 @@ def _parse_string(self, name):
 
                     season_number = int(tv_episode['seasonnumber'])
                     episode_numbers = [int(tv_episode['episodenumber'])]
-                    logger.debug('Indexer info for show {name}: {ep}',
-                                 name=result.show.name, ep=episode_num(season_number, episode_numbers[0]))
+                    log.debug(
+                        'Indexer info for series {name}: {ep}', {
+                            'name': result.show.name,
+                            'ep': episode_num(season_number, episode_numbers[0]),
+                        }
+                    )
                 except IndexerEpisodeNotFound:
-                    logger.warning("Unable to find episode with date {date} for show '{name}'. Skipping",
-                                   date=result.air_date, name=result.show.name)
+                    log.warning(
+                        'Unable to find episode with date {date} for series {name}. Skipping',
+                        {'date': result.air_date, 'name': result.show.name}
+                    )
                     episode_numbers = []
-                except IndexerError as e:
-                    logger.warning('Unable to contact {indexer_api.name}: {e}', indexer_api=indexer_api, e=e.message)
+                except IndexerError as error:
+                    log.warning(
+                        'Unable to contact {indexer_api.name}: {error}',
+                        {'indexer_api': indexer_api, 'error': error.message}
+                    )
                     episode_numbers = []
-                except IndexerException as e:
```

Comment:
@fernandog my second comment to @ratoaq2 is exactly why names like `e` are bad...  its too easy to end up clobbering an existing variable and its not descriptive of the content... just like `e` for episode is bad too.